terraform/main.tf

```diff
@@ -2,224 +2,84 @@ provider "aws" {
+  sgtm_feature__check_rerun_base_ref_names       = var.sgtm_feature__check_rerun_base_ref_names
+  sgtm_feature__check_rerun_on_approval_enabled  = var.sgtm_feature__check_rerun_on_approval_enabled
+  sgtm_feature__check_rerun_threshold_hours      = var.sgtm_feature__check_rerun_threshold_hours
+  sgtm_feature__disable_github_team_subscription = var.sgtm_feature__disable_github_team_subscription
+  sgtm_feature__followup_review_github_users     = var.sgtm_feature__followup_review_github_users
+  sgtm_rest_api_id                               = aws_api_gateway_rest_api.sgtm_rest_api.id
+  sgtm_rest_api_root_resource_id                 = aws_api_gateway_rest_api.sgtm_rest_api.root_resource_id
+  sgtm_rest_api_execution_arn                    = aws_api_gateway_rest_api.sgtm_rest_api.execution_arn
+}
+
+
+module "sgtm-staging" {
+  source                                         = "./sgtm-cluster"
+  naming_suffix                                  = "staging"
+  api_encryption_key_arn                         = aws_kms_key.api_encryption_key.arn
+  api_key_s3_bucket_name                         = var.api_key_s3_bucket_name
+  api_key_s3_object                              = var.api_key_s3_object
+  dynamodb-sgtm-lock-arn                         = aws_dynamodb_table.sgtm-lock.arn
+  dynamodb-sgtm-objects-arn                      = aws_dynamodb_table.sgtm-objects.arn
+  github_usernames_to_asana_gids_s3_path         = var.github_usernames_to_asana_gids_s3_path
+  lambda_code_s3_bucket_name                     = var.lambda_code_s3_bucket_name
+  lambda_function_timeout                        = var.lambda_function_timeout
+  lambda_runtime                                 = var.lambda_runtime
+  s3_api_key_bucket_arn                          = aws_s3_bucket.api_key_bucket.arn
+  sgtm_feature__allow_persistent_task_assignee   = var.sgtm_feature__allow_persistent_task_assignee
+  sgtm_feature__autocomplete_enabled             = var.sgtm_feature__autocomplete_enabled
+  sgtm_feature__automerge_enabled                = var.sgtm_feature__automerge_enabled
+  sgtm_feature__check_rerun_base_ref_names       = var.sgtm_feature__check_rerun_base_ref_names
+  sgtm_feature__check_rerun_on_approval_enabled  = var.sgtm_feature__check_rerun_on_approval_enabled
+  sgtm_feature__check_rerun_threshold_hours      = var.sgtm_feature__check_rerun_threshold_hours
+  sgtm_feature__disable_github_team_subscription = var.sgtm_feature__disable_github_team_subscription
+  sgtm_feature__followup_review_github_users     = var.sgtm_feature__followup_review_github_users
+  sgtm_rest_api_id                               = aws_api_gateway_rest_api.sgtm_rest_api.id
+  sgtm_rest_api_root_resource_id                 = aws_api_gateway_rest_api.sgtm_rest_api.root_resource_id
+  sgtm_rest_api_execution_arn                    = aws_api_gateway_rest_api.sgtm_rest_api.execution_arn
+}
+
+module "sgtm-staging-sqs" {
```

Comment:
https://github.com/Asana/SGTM/pull/173 adds a new staging cluster for sqs testing so we need to track it here for terraform to work properly

Reply:
this was later removed in that same PR right? did you still need it in this PR?

Reply:
yep will delete this staging cluster and remove it here